docs/ref/contrib/postgres/operations.txt

```diff
@@ -30,11 +30,12 @@ For example::
             ...
         ]
 
-Creating the extension requires a database user with superuser privileges.
-If the Django database user doesn't have superuser privileges, you'll have
-to create the extension outside of Django migrations with a user that has
-the appropriate privileges. In that case, connect to your Django database and
-run the query ``CREATE EXTENSION IF NOT EXISTS hstore;``.
+Most of the time creating the extension requires a database user with the
+``SUPERUSER`` privilege.  If the Django database user doesn't have that
+privilege, the extension will have to be created outside Django migrations by
+running the query ``CREATE EXTENSION hstore;``. In case the extension is
+already present in the database, the operation requiring ``SUPERUSER`` will be
+skipped.
```

Comment:
`versionchanged` annotation is missing.

Reply:
i consider this a bug fix, not a new feature. Django claims to work fine with non-superuser db users, but that's not true atm.

Reply:
You want to change the current behavior so we need a `versionchanged`. Added note is not true in older versions of Django.

Reply:
which version number would i put there?

Reply:
3.2